plugins/powerd/fu-plugin-powerd.c

```diff
@@ -69,6 +96,11 @@ fu_plugin_startup(FuPlugin *plugin, GError **error)
 {
 	FuPluginData *data = fu_plugin_get_data(plugin);
 	g_autofree gchar *name_owner = NULL;
+	gboolean file_absent = fu_plugin_powerd_create_suspend_file(error);
+	if (!file_absent) {
+		g_prefix_error(error, "lock file unable to be deleted");
```

Comment:
Why do we prefix the error here and not in cleanup?

Reply:
We could have it as:
```
if (!fu_plugin_powerd_delete_suspend_file(error))
    return FALSE;
```

Reply:
I moved the error to the function itself, and added one in for create (didn't see the suggestion until just now). How do you feel about having it that way?